gputop-server/gputop-server.c

```diff
@@ -973,6 +972,71 @@ gputop_get_cmd_line_pid(uint32_t pid, char *buf, int len)
     return res;
 }
 
+void  handle_update_hw_id_map(uint64_t *vgpu_id,
```

Comment:
This function should be static.
You can remove the double space after void.

Reply:
Thanks, I have corrected it.